guides/source/rails_application_templates.md

```diff
@@ -259,9 +252,7 @@ are generated. Useful for all generated files to version control:
 
 ```ruby
 after_bundle do
-  git :init
-  git add: '.'
-  git commit: "-a -m 'Initial commit'"
+  rails_command "webpacker:install"
```

Comment:
Instead of repeating the git example again, I choose a new example, that also illustrates the `....:install` command. There's no example of this in the other examples.